src/main/java/com/opencart/pages/CategorieContainer.java

```diff
@@ -0,0 +1,29 @@
+package com.opencart.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class CategorieContainer {
```

Comment:
Could you, please, explain, why do you need CategorieContainer class?

Reply:
Actually, I thought that I could use it when I click on the select Category button, but further, I didn't use this class.

Reply:
Okay, so, please, remove it

Reply:
Ok